OpenEdXMobile/res/values/strings.xml

```diff
@@ -662,4 +662,6 @@
     <string name="rating_bar_selection">{rating} out of {num_of_stars}</string>
     <!-- Title of the Whats New screen e.g. Version 2.8.1 New Features -->
     <string name="whats_new_title">Version {version_number} New Features</string>
+    <!-- Text message if permisssion not granted -->
+    <string name="permission_not_granted">Permission not granted</string>
```

Comment:
`tools:ignore="MissingTranslation"` needs to be added so that lint error can be catered.